v5.0.0-alpha.36 (#26636)
mui-org/material-ui: CHANGELOG.md

### [Versions](https://material-ui.com/versions/)

## 5.0.0-alpha.36

<!-- generated comparing v5.0.0-alpha.35..next -->

_Jun 7, 2021_

Big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:

> We've introduced a new `ListItemButton` component that should prevent common mistakes when using `<ListItem button />` and help with customization and TypeScript support.
> 
> `experimentalStyled` is now available without the `experimental` prefix.
> We're confident that it's API shouldn't receive any major changes until the stable release of v5.
> 
> `@material-ui/icons` only ships ES modules and no longer CommonJS modules.
> This reduces the download size of the package from 1.7 MB to 1.2 MB and install size from 15.6 MB to 6.7 MB.

Reviewer: ```suggestion
- ⚒️ We've introduced a new `ListItemButton` component that should prevent common mistakes when using `<ListItem button />` and help with customization and TypeScript support.
- 👩‍🎤 `experimentalStyled` is now available without the `experimental` prefix.
We're confident that it's API shouldn't receive any major changes until the stable release of v5.
- 📦 `@material-ui/icons` only ships ES modules and no longer CommonJS modules.
This reduces the download size of the package from 1.7 MB to 1.2 MB and install size from 15.6 MB to 6.7 MB.
```

To be closer to previous releases' description

Reviewer 2: How about we also link the PR so developers can dive if needed?

```suggestion
- ⚒️ We've introduced a new `ListItemButton` component that should prevent common mistakes when using `<ListItem button />` and help with customization and TypeScript support (#26446) @siriwatknp.
- 👩‍🎤 `experimentalStyled` is now available without the `experimental` prefix.
We're confident that its API shouldn't receive any major changes until the stable release of v5 (#26558) @mnajdova.
- 📦 `@material-ui/icons` only ships ES modules and no longer CommonJS modules.
This reduces the download size of the package from 1.7 MB to 1.2 MB and installs size from 15.6 MB to 6.7 MB (#26310) @eps1lon.
```

Reviewer 2: I think that we should talk about #26559 too, its issue had over 30 upvotes

### `@material-ui/core@5.0.0-alpha.36`

#### Breaking changes

- <!-- 10 --> [AppBar] Fix background color on dark mode (#26545) @siriwatknp

> - <!-- 13 --> [core] Rename `experimentalStyled` to `styled` (#26558)

Reviewer 2: ```suggestion
- <!-- 13 --> [core] Rename `experimentalStyled` to `styled` (#26558) @mnajdova
```

  Remove the experimental prefix, this module is going stable:

  ```diff
  -import { experimentalStyled as styled } from '@material-ui/core/styles';
  +import { styled } from '@material-ui/core/styles';
  ```

> - <!-- 03 --> [SwitchBase] replace IconButton with ButtonBase (#26460)

Reviewer 2: ```suggestion
- <!-- 03 --> [SwitchBase] replace IconButton with ButtonBase (#26460) @siriwatknp
```

- <!-- 25 --> [theme] Improve default `primary`, `secondary` and `error` theme palette (#26555) @siriwatknp

#### Changes

- <!-- 17 --> [Box] Fix module 'clsx' not found in system (#26553) @coder-freestyle
- <!-- 07 --> [Box] Fix runtime error when using styled-components without ThemeProvider (#26548) @mnajdova

> - <!-- 27 --> [Radio][Checkbox] Don't forward `color` to DOM elements (#26625) @siriwatknp

Reviewer 2: Maybe to provide a bit more context? (it's a regression)

```suggestion
- <!-- 27 --> [Radio][Checkbox] Fix forward `color` to DOM elements regression (#26625) @siriwatknp
```

Author: Why is that important?

Reviewer 2: Ok, resolving

  This should not affect you if you're using a bundler like `webpack` or `snowpack` or meta frameworks like `next` or `gatsby`.

### `@material-ui/lab@5.0.0-alpha.36`

#### Breaking changes

- <!-- 29 --> [pickers] Remove `openPickerIcon` prop in favor of `components.OpenPickerIcon` (#26223) @vedadeepta

### `@material-ui/system@5.0.0-alpha.36`

- <!-- 18 --> [system] Add createTheme util (#26490) @mnajdova

### Docs

- <!-- 28 --> [docs] Migrate templates to emotion (#26604) @vicasas
- <!-- 26 --> [docs] Remove custom primary & secondary color (#26541) @siriwatknp
- <!-- 24 --> [docs] Don't escape prop descriptions for markdown table context (#26579) @eps1lon
- <!-- 22 --> [docs] Prepare for data grid auto-generated docs (#26477) @m4theushw
- <!-- 21 --> [docs] Fix typo sx !== xs (#26596) @onpaws
- <!-- 16 --> [docs] Multiple select demos moving when selecting values (#26539) @itsnorbertkalacska
- <!-- 14 --> [docs] Improve migration guide for `@material-ui/styles` (#26552) @mnajdova
- <!-- 12 --> [docs] `Rating` `value` is nullable in `onChange` (#26542) @sakura90
- <!-- 30 --> [example] Remove the dependency on @material-ui/styles (#26567) @garfunkelvila

### Core

> @mnajdova
> 

Reviewer: ```suggestion
```

Remove me 😄
